### DISCUSSION TITLE
Making synthesis with a VITs voice on CPU faster

### DISCUSSION THREAD

**Discussion by NoteToSelfFindGoodNickname** [coqui-ai/TTS#1884] [General]
Hello!

Would it be possible to speed up the synthesis a lot so that synthesizing a sentence with VITs on CPU would take only around 200 ms?

Currently synthesizing a sentence with a VITs voice on CPU like "This is a test just to hear what this voice would sound." takes 2 seconds on a desktop computer with a good GPU.

Traditional TTS system using unit concatenation only need like 50 ms for a sentence. This is what I am looking for ideally. :-)

**Comment by domcross** [comment]
Solution: buy faster hardware 😉 

As usual there is a tradeoff, here it is quality vs speed. To my knowledge (and experience) VITS is among the models that meet the optimum between speech quality and inference speed.

Are you already using the [TTS server](https://github.com/coqui-ai/TTS/tree/dev/TTS/server)? If not you should try it, it saves you the time spent for loading the model for each inference call.

**Comment by NoteToSelfFindGoodNickname** [comment]
Thank you, but I am looking for an offline solution without a server.

**Reply by domcross** [reply]
TTS-server is a piece of software that you can run on the same machine where you run the inference command.
Maybe you should specify your use-case in more detail...

**Reply by NoteToSelfFindGoodNickname** [reply]
You recommend the TTS server because it initializes the model only once, correct?
I would like to use the voice in a software as a screen reader. For this purpose, I am aiming at around 200 ms for a sentence or less.

**Reply by thorstenMueller** [reply]
I'm not @domcross , but yes - using these TTS server will just load model once. But honestly i doubt that 200ms is realistic using Coqui TTS at this time (obviously depending on your hardware).
